frontend/src/styles.scss

```diff
@@ -79,7 +79,7 @@ body {
   }
 
     app-main,
-    app-example1{
+    app-example1 {
```

Comment:
skad ta 1 ?

Reply:
eee... co? przecież to komponent tak sie nazywa: "example1" O.o

Reply:
dlaczego example 1? Macie ich wiecej? :D

Reply:
no byly 2 :D / zrobic cos z tym?

Reply:
Zrob :D